README.adoc

```diff
@@ -182,24 +186,22 @@ Note that, when the value of the gauge is retrieved, the underlying `getTotal()`
 called to return the value.
 
 These annotations are registered by default to the `MetricRegistry` singleton object.
-Through this object, they send metadata and metrics to the
-`https://localhost:9443/metrics/application` endpoint, which is provided for you. Also, the
-`@Counted`, `@Gauge`, and `@Timed` annotations can be placed in the Resource, such as
-`InventoryResource`.
+This object sends the metadata and metrics to the `https://localhost:9443/metrics/application` 
+MicroProfile Metrics endpoint, which is provided for you. Also, the `@Counted`, `@Gauge` and 
+`@Timed` annotations can be placed in a service, such as the `inventory` service in the 
+`src/main/java/io/openliberty/guides/inventory/InventoryResource.java` file.
 
 // =================================================================================================
 // Building and running the application
 // =================================================================================================
 
 include::{common-includes}/mvnbuild.adoc[]
 
-When the server runs, navigate to the following URL to find the metrics endpoint that shows JVM and server base
-details:
+Point your browser to the `https://localhost:9443/metrics` URL to find the metrics that shows JVM 
+and server base details. 
```

Comment:
I don't think it matters, but in case it does, I also think we have an extra space after "details."

Reply:
lol.  I fixed